src/section/member_info.rs

```diff
@@ -94,7 +94,10 @@ impl MemberInfo {
         self.age_counter >= AgeCounter(2_u32.pow(MAX_INFANT_AGE + 1))
     }
 
-    #[cfg(feature = "mock_base")]
+    pub fn is_new_adult(&self) -> bool {
+        self.age_counter_value() == MAX_INFANT_AGE + 1
```

Comment:
self.age_counter_value() is actually the real number of churned times, which is `2's power of age`.
I think the function here needs to be :
`self.age_counter_value() == 2_u32.saturating_pow(u32::from(MAX_INFANT_AGE + 1))`